.github/workflows/test-warehouse.yml

```diff
@@ -130,6 +123,15 @@ jobs:
           pip install -r dev-requirements.txt
           pip install ".[${{ (inputs.warehouse-type == 'databricks_catalog' && 'databricks') || inputs.warehouse-type }}]"
 
+      - name: Write dbt profiles
+        env:
+          PROFILES_YML: ${{ secrets.CI_PROFILES_YML }}
+        run: |
+          mkdir -p ~/.dbt
+          DBT_VERSION=$(pip show dbt-core | grep -i version | awk '{print $2}' | sed 's/\.//g')
```

Comment:
Should this be `inputs.dbt-version` because this is what defines the concurrency group?

Reply:
Not sure I understood the quesetion

Reply:
Oh if you're asking if you should use the version from the inputs that I think you should

Reply:
It's just a bit annoying because half of the time it'll be empty so I prefer to remain consistent and always have the dbt version in the schema name.